### PR TITLE
added german language files

### PR DESCRIPTION
**Pull request by tomxpcvx**
## Context 

* This PR adds german language support for the docusaurus.

## Choices

* 

## Test instructions

1. Navigate to the docusaurus folder
2. Run: `npm start -- --locale de`

## Checklist before requesting a review

* [x] I have performed a self-review of my code
* [x] I've added documentation about this change to the README.
* [x] I've not introduced breaking changes.

### CODE REVIEW

**Comment by tomxpcvx** [comment]
Hi Thijs,
is it also possible to translate the content from the Markdown files? 😄

**Comment by tomxpcvx** [comment]
Hi :smile:
I just wanted to make it clear that I've started translating, not that several people are making the effort at the same time and can translate together instead.

**Comment by MSpreckels** [comment]
you might want to mark it as draft if you dont want @thijsvanloef to merge it :)

**Comment by tomxpcvx** [comment]
Hey guys 👋🏼
I squashed my commits into one commit, so that everything is easier to maintain 😄 

I also checked that everything looks good in docusaurus and checked the links pointing to headings hopefully work in production. I used the Dutch language files as a guide because it works there.

In the meantime there have been markdown lint checks. Unfortunately I can't see the current status.

If there is anything that needs to be fixed, please let me know. 😊

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/advanced/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+* `kubectl apply -f pvc.yaml`
+* `kubectl apply -f configmap.yaml`
+* `kubectl apply -f secret.yaml`
+* `kubectl apply -f service.yaml`
+* `kubectl apply -f deployment.yaml`
+
+## Verwendung des Helm-Charts
+
+Der offizielle Helm-Chart befindet sich in einem separaten Repository, [palworld-server-chart](https://github.com/Twinki14/palworld-server-chart)
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/advanced/kubernetes.md:21]: Das offizielle Helm-Chart [...]

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/advanced/palworld-directory.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Alles was mit den Palworld-Dateien zusammenhängt, befindet sich im Ordner `/palworld` innerhalb des Containers.
+
+## Ordnerstruktur
+
+![Ordnerstruktur](../assets/folder_structure.jpg)
+
+| Ordner                       | Verwendung                                                             |
+|------------------------------|------------------------------------------------------------------------|
+| palworld                     | Stammordner mit allen Dateien des Palworld-Servers                     |
+| backups                      | Ordner, in dem alle Backups des Befehls `backup` gespeichert werden    |
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/advanced/palworld-directory.md:16]: Reicht es nicht zu sagen

"Der Ordner, in dem alle Backups gespeichert werden" ?

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+## Mit Umgebungsvariablen
+
+:::warning
+Diese Umgebungsvariablen und Einstellungen können sich ändern, da das Spiel noch in der Beta-Phase ist.
+
+Überprüfen Sie die [offizielle Webseite für unterstützte Parameter.](https://tech.palworldgame.com/optimize-game-balance)
+:::
+
+Das Konvertieren von Servereinstellungen in Umgebungsvariablen folgt denselben Prinzipien (mit einigen Ausnahmen):
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md:17]: Das liest sich komisch. Vllt sowas wie:

"Um Servereinstellung in Umgebungsvariablen zu konvertieren müssen folgende Regeln eingehalten werden:"

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+* alle Buchstaben groß schreiben
+* Wörter durch Einfügen eines Unterstrichs trennen
+* den einzelnen Buchstaben entfernen, wenn die Einstellung damit beginnt (wie 'b')
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md:21]: "falls die Einstellung mit einem einzelnen Buchstaben beginnt, muss dieser entfernt werden (z.B. 'b')."

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+| DAYTIME_SPEEDRATE                         | Tageszeit-Geschwindigkeit - Kleinerer Wert bedeutet kürzere Tage | 1.000000                                                                                     | Float                                  |
+| NIGHTTIME_SPEEDRATE                       | Nachtszeit-Geschwindigkeit - Kleinerer Wert bedeutet kürzere Nächte | 1.000000                                                                                     | Float                                  |
+| EXP_RATE                                  | EXP-Sammelrate                                                 | 1.000000                                                                                     | Float                                  |
+| PAL_CAPTURE_RATE                          | Pal-Fangerate                                                  | 1.000000                                                                                     | Float                                  |
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md:35]: Da ist ein 'e' zu viel oder nicht?
"Pal-Fangrate"

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+| PAL_SPAWN_NUM_RATE                        | Pal-Erscheinungsrate                                           | 1.000000                                                                                     | Float                                  |
+| PAL_DAMAGE_RATE_ATTACK                    | Schadensmultiplikator von Pals                                | 1.000000                                                                                     | Float                                  |
+| PAL_DAMAGE_RATE_DEFENSE                   | Schadensmultiplikator zu Pals                                 | 1.000000                                                                                     | Float                                  |
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md:38]: "Verteidigungsmultiplikator von Pals"

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+| PLAYER_DAMAGE_RATE_ATTACK                 | Schadensmultiplikator von Spielern                            | 1.000000                                                                                     | Float                                  |
+| PLAYER_DAMAGE_RATE_DEFENSE                | Schadensmultiplikator zu Spielern                              | 1.000000                                                                                     | Float                                  |
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md:40]: Verteidigungsmultiplikator [...]

File: docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/quick-setup.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+<!-- markdownlint-disable-next-line -->
+Ändern Sie Ihre [docker-compose.yml](https://github.com/thijsvanloef/palworld-server-docker/blob/main/docker-compose.yml) wie folgt:
+
+```yml
+services:
+   palworld:
+      image: thijsvanloef/palworld-server-docker:latest # Verwenden Sie das Tag latest-arm64 für ARM64-Hosts
+      restart: unless-stopped
+      container_name: palworld-server
+      stop_grace_period: 30s # Auf die Zeit festlegen, die Sie bereit sind zu warten, bis der Container ordnungsgemäß beendet ist
```
Comment (MSpreckels) [inline docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/quick-setup.md:75]: "die Zeit, die Sie bereit sind zu warten, bis der Container ordnungsgemäß beendet ist"

**Review by MSpreckels — CHANGES_REQUESTED** [review]
Hey there! I've skimmed over the translation and found a few things. IMO, it's an accurate translation which sometimes feels abit stiff.

**Comment by tomxpcvx** [comment]
Hi, I have added your suggestions. I also tried not to change too much of the meaning and form of the original texts. And in some places it was actually a bit stiff.

**Pull request merged**